packages/core/data_adapters/BaseAdapter.ts

```diff
@@ -93,7 +93,7 @@ export abstract class BaseAdapter {
   public abstract freeResources(region: Region): void
 }
 
-export interface Stats {
+export interface FeatureDensityStats {
```

Comment:
this could potentially be changed to e.g. {featureDensity:number} | {bytes:number,fetchSizeLimit:number} but that requires additional code changes to make typescript happy, and i think it's ok to leave it as is for now